cardano-sync/src/Cardano/Sync.hs

```diff
@@ -146,8 +152,15 @@ runSyncNode dataLayer plugin enp insertValidateGenesisDist =
       case genCfg of
           GenesisCardano _ bCfg _sCfg -> do
             syncEnv <- ExceptT $ mkDbSyncEnvFromConfig dataLayer (enpLedgerStateDir enp) genCfg
-            liftIO $ runDbSyncNodeNodeClient (dncPrometheusPort enc) syncEnv
-                iomgr trce plugin (cardanoCodecConfig bCfg) (enpSocketPath enp)
+            liftIO $ runDbSyncNodeNodeClient
```

Comment:
Why change the layout style from what was there. A good rule of thumb when submitting a PR to a project is to follow the existing coding style, eg "When in Rome do as the Romans".

Reply:
Fine, I can revert that.